CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changes by Version
 ==================
 
+0.43.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.43.0
```

Comment:
Since there is 0.43.1 we should use that directly.

Reply:
The image and binaries should be at 0.43.0, even though a 0.43.1 tag exists for the core repo.

Reply:
ok, then I should keep 0.43.0, right?

Reply:
Yes, to end-users, there's only 0.43.0. If we use APIs from the core, the go.mod will have references to 0.43.1.

Reply:
oh I see, if there was new container release https://github.com/open-telemetry/opentelemetry-collector-releases/releases we should use that.